lib/rubocop/cop/style/block_delimiters.rb

```diff
@@ -369,9 +370,8 @@ def get_blocks(node, &block)
         end
         # rubocop:enable Metrics/CyclomaticComplexity
 
-        # rubocop:disable Metrics/CyclomaticComplexity
         def proper_block_style?(node)
-          return true if require_braces?(node) || require_do_end?(node)
+          return true if require_do_end?(node)
```

Comment:
The changes in 1b237c643c9583846b4599b4c5ba32411a29091d mean that the `require_braces?` method is now unnecessary, as any code that would cause it return `true` is already ignored by `on_send` above. As such, I removed it.

This can be verified by making this removal in `master` and observing that no tests fail.